flatdict.py

```diff
@@ -487,4 +483,8 @@ def _child_as_list(self, pk, ck=None):
                 elif subset[k].original_type == dict:
                     out.append(subset[k].as_dict())
             return out
-        return [subset[k] for k in sorted(keys, key=lambda x: int(x))]
+
+        if sys.version_info[0:2] <= (3, 5):
```

Comment:
Why version specific sorting here?

Reply:
This was due to 3.4, since we can drop it I'll remove it.

Reply:
Actually that also affects python 3.5, but not 3.6+ since we have the new dict that ensure insertion order, so the check is mostly for "improve" performance on 3.6+ for huge lists inside dicts, what do you want here?, it's okay to leave with a comment ?

Reply:
Leave with comment to remove after 3.6, perhaps change it to < (3, 6) just to be more explicit about it.